src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -2668,7 +2668,7 @@ Address MacroAssembler::spill_address(int size, int offset, Register tmp)
   if ((offset & (size-1)) && offset >= (1<<8)) {
     add(tmp, base, offset & ((1<<12)-1));
     base = tmp;
-    offset &= -1<<12;
+    offset &= -(1<<12);
```

Comment:
Note, this change differs from the upstream one and is not equivalent. See:
http://hg.openjdk.java.net/jdk/jdk/rev/e53ec3b362f4